main.nf

```diff
@@ -618,6 +618,7 @@ process Get_software_versions {
     trim_galore -v &> v_trim_galore.txt 2>&1 || true
     vcftools --version &> v_vcftools.txt 2>&1 || true
     vep --help &> v_vep.txt 2>&1 || true
+    msisensor &> v_msisensor.txt 2>&1 || true
```

Comment:
Can you also update the regexes in `bin/scrape_software_versions.py` to match the msisensor output?
https://github.com/nf-core/sarek/blob/52bee464a89b1a891d608637b67f5c9a1da8aead/bin/scrape_software_versions.py#L6

Reply:
done in e0af15e